app/views/licences/tabs/communications.njk

```diff
@@ -5,9 +5,15 @@
   {% if communication.type.pdf %}
     <p class="govuk-body govuk-!-margin-bottom-0">{{ communication.type.label }}<span class="govuk-visually-hidden"> {{ communication.type.sentVia }}</span></p>
   {% else %}
-    <a href="/licences/{{ documentId }}/communications/{{ communication.id }}" class="govuk-link">{{ communication.type.label }}
-      <span class="govuk-visually-hidden"> {{ communication.type.sentVia }}</span>
-    </a>
+    {% if enableNotificationsView %}
+      <a href="/system/notifications/{{ communication.id }}?id={{ licenceId }}" class="govuk-link">{{ communication.type.label }}
+        <span class="govuk-visually-hidden"> {{ communication.type.sentVia }}</span>
+      </a>
+    {% else %}
+      <a href="/licences/{{ documentId }}/communications/{{ communication.id }}" class="govuk-link">{{ communication.type.label }}
+        <span class="govuk-visually-hidden"> {{ communication.type.sentVia }}</span>
+      </a>
+    {% endif %}
```

Comment:
I think it would be better to determine the link to use in the view-licence-communications presenter. Doing it there means we can also write unit tests to ensure it is being defined correctly.